drivers/rf/cc1101/cc1101.h

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2021 Leonard Pollak
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_RF_CC1101_H_
+#define ZEPHYR_DRIVERS_RF_CC1101_H_
+
+#include "cc1101_registers.h"
+
+/* R/W Bits */
+#define CC1101_SPI_WRITE	0x00
+#define CC1101_SPI_BURST_WRITE	0x40
+#define CC1101_SPI_READ		0x80
+#define CC1101_SPI_BURST_READ	0xC0
+
+#define CC1101_FIFO_SIZE	64U
+#define CC1101_FIFO_HWM		32U
+#define CC1101_IOCFG_CHIP_RDY	0x29
+#define CC1101_IOCFG_RX		0x00
+#define CC1101_IOCFG_TX		0x02
+#define CC1101_IOCFG_XFER	0x06
+#define CC1101_IOCFG_HIZ	0x2E
+
+#define CC1101_NUM_CFG_REG	47U
+#define CC1101_NUM_PATABLE	8U
+
+#define CC1101_ALLOWED_PKT_LEN_MODE	0x01
+#define CC1101_ALLOWED_PKT_FORMAT	0x00
+
+union cc1101_data_item {
+        uint8_t buffer[CONFIG_CC1101_MAX_PACKET_SIZE];
+        struct {
+                uint8_t length;
+                uint8_t payload[CONFIG_CC1101_MAX_PACKET_SIZE - 1];
+        } __packed;
+};
+
+char __aligned(4) rxq_buffer[CONFIG_CC1101_RX_MSG_QUEUE_DEPTH * sizeof(union cc1101_data_item)];
+
+union CC1101_CHIP_STATUS {
+	uint8_t reg;
+	struct {
+		uint8_t FIFO_BYTES_AVAILABLE: 4;
```

Comment:
Use small letters for attribute names. 

Capital letters mostly go only to macros and enums.

Apply that everywhere.